### ISSUE TITLE
Object destructuring and imports fail object-curly-newline rules

### ISSUE THREAD

**Issue by rmccue** [humanmade/coding-standards#58]
The following are currently failing the `object-curly-newline` rules, and should not be:

```js
import { Route, withRouter } from 'react-router-dom';
const { lastView, onDismiss } = props;
```

These appear to now have enforced newlines, when they should be optional.

**Comment by rmccue** [comment]
Also:
```js
renderItem: ( { item, selected, onSelect } ) => {
```

**Comment by rmccue** [comment]
Also, maybe:
```js
const fillProps = { author, comments, categories, post };
```
